tests/ut/ut_scargo_fix.py

```diff
@@ -0,0 +1,44 @@
+from unittest.mock import MagicMock
+
+import pytest
+from pytest_mock import MockerFixture
+
+from scargo.commands.fix import scargo_fix
+from scargo.config import Config
+
+
+def test_scargo_fix_pragma(
+    mock_prepare_config: Config,
+    caplog: pytest.LogCaptureFixture,
+) -> None:
+    scargo_fix(True, True, True)
+    assert "Fixed" in caplog.text
+
+
+def test_scargo_fix_copyright(
+    mock_prepare_config: Config,
+    caplog: pytest.LogCaptureFixture,
+) -> None:
+    scargo_fix(True, True, True)
+    assert "Fixed" in caplog.text
+
+
+def test_scargo_fix_clang_format(
+    mock_prepare_config: Config,
+    caplog: pytest.LogCaptureFixture,
+) -> None:
+    scargo_fix(True, True, True)
+    assert "Fixed" in caplog.text
```

Comment:
All three tests are exactly the same.
There are no keywords, so the `True` and `False` arguments seem cryptic.
Is this assertion enough? a `print("Fixed")` function would pass these tests.

Reply:
> There are no keywords, so the True and False arguments seem cryptic.

This is still the case. However, I'm no longer on this project so feel free to resolve this thread whenever you want.